lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -258,6 +258,8 @@ class CosAgentProviderUnitData(pydantic.BaseModel):
     metrics_alert_rules: dict
     log_alert_rules: dict
     dashboards: List[GrafanaDashboard]
+    # subordinate is no longer used but we should keep it until we bump the library to ensure
```

Comment:
could make this clear by adding a `__post_init__` or validator and, if the field is provided (non-defaulted), pop a warning.

Reply:
Probably not needed as it works perfectly well and will fix itself on a lib upgrade.